src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -817,7 +821,7 @@ private void CheckDisposed()
                     finally
                     {
                         // The buffer was sliced up to where it was consumed, so we can just advance to the start.
-                        // We mark examined as buffer.End so that if we didn't receive a full frame, we'll wait for more data
+                        // We mark examined as buffer. End so that if we didn't receive a full frame, we'll wait for more data
```

Comment:
We actually don't want a space here since `buffer.End` is a property, so the period isn't ending the sentence ;)